src/lxc/tools/lxc_unshare.c

```diff
@@ -41,10 +41,112 @@
 
 #include "arguments.h"
 #include "caps.h"
-#include "conf.h"
+#include "log.h"
 #include "namespace.h"
 #include "utils.h"
 
+lxc_log_define(lxc_unshare, lxc);
+
+struct start_arg {
+	char *const *args;
+	int flags;
+	uid_t uid;
+	bool setuid;
+	int want_default_mounts;
+	int wait_fd;
+	const char *want_hostname;
+};
+
+struct my_iflist
+{
+	char *mi_ifname;
+	struct my_iflist *mi_next;
+};
+
+static int my_parser(struct lxc_arguments *args, int c, char *arg);
+static inline int sethostname_including_android(const char *name, size_t len);
+static int get_namespace_flags(char *namespaces);
+static bool lookup_user(const char *optarg, uid_t *uid);
+static int mount_fs(const char *source, const char *target, const char *type);
+static void lxc_setup_fs(void);
+static int do_start(void *arg);
+
+static struct my_iflist *tmpif, *my_iflist = NULL;
```

Comment:
No need to initialize the variables. They are global and are thus initialized to zero anyway. :)

Reply:
@brauner Right! Thanks.